specTemplate.spec

```diff
@@ -17,6 +17,8 @@
 
 %define _unpackaged_files_terminate_build 0
 
+%define _is_arm __is_arm__
```

Comment:
Prefer (for flexibility)

```spec
# Do not check any files in lib/arm for requires
%global __requires_exclude_from ^%{_prefix}/lib/arm/.*$
%global __requires_exclude_from ^%{_prefix}/lib/arm/*.*$
# Do not check any files in bin/arm for requires
%global __requires_exclude_from ^%{_prefix}/bin/arm/.*$
%global __requires_exclude_from ^%{_prefix}/bin/arm/*.*$
# Do not check .so files in an arm-specific library directory for provides
%global __provides_exclude_from ^%{_prefix}/lib/arm/*\\.so$
%global __provides_exclude_from ^%{_prefix}/lib/arm/*\.so$

%define add_arm_libs %( if [ -d 'lib/arm' ]; then echo "1" ; else echo "0"; fi )
%define is_arm  %( if [[ '__buildarch__' =~ "arm" ]]; then echo "1" ; else echo "0"; fi )
%define not_arm  %( if [[ ! '__buildarch__' =~ "arm" ]]; then echo "1" ; else echo "0"; fi )
```

Reply:
I don't see how this will improve the matter if things...

Reply:
The spec macros eliminate the need to define a variable in the `mfRPM.mk` file that is never used there, and allow the packaging of an `arm` library in a `devel` package (needed)

The other macros make sure that these packaged `arm` libs don't cause a failed installation due to requirements/provision checking in the non-`arm` package

Reply:
Ups, missed it was _already_ there. Extra var removed, code updated